Source/Scene/Scene.js

```diff
@@ -1241,7 +1241,7 @@ define([
                     // When moving the camera low LOD terrain tiles begin to load, whose bounding volumes
                     // throw off the near/far fitting for the shadow map. Only update shadowNear and shadowFar
                     // for reasonably sized bounding volumes.
-                    if (!(distances.start < 0.0 && distances.stop > frameState.shadowFar)) {
+                    if (!((distances.start < -100.0) && (distances.stop > 100.0) && (pass === Pass.GLOBE))){
```

Comment:
Might want to add a `TODO` that 3D Tiles are not part of `Pass.GLOBE`.  However, we can introduce `Pass.ENVIRONMENT`, etc., but will need to think through opaque vs. translucent.